### ISSUE TITLE
[Marketing] Add Fasten to Awesome* lists

### ISSUE THREAD

**Issue by AnalogJ** [fastenhealth/fasten-onprem#155]
- https://github.com/kakoni/awesome-healthcare#readme
- https://github.com/awesome-selfhosted/awesome-selfhosted
- https://github.com/awesome-foss/awesome-sysadmin

**Comment by justinKocian** [comment]
This seems easy enough - I’ll see if I can get this done over the next few days.

**Comment by justinKocian** [comment]
Not sure if this would be a good fit for awesome-sysadmin, as it’s meant to be for personal records. Thoughts?

**Comment by AnalogJ** [comment]
Yeah, good point. We should probably skip that one. I just did a cursory search of awesome lists that were relevant, there's probably a couple that I missed.

**Comment by AnalogJ** [comment]
@justinKocian if you open PR's can you link them to this issue just so we don't get any duplication? 

Thanks!

**Comment by justinKocian** [comment]
Yeah of course! I’ll get started tomorrow when I’m at my desk.

**Comment by cfu288** [comment]
@AnalogJ Let me know if you want to use another description for Fasten.

**Comment by AnalogJ** [comment]
oh that's awesome! 😉 
Thanks for doing this @cfu288 !

**Comment by haoyu-haoyu** [comment]
I've submitted a PR to add Fasten Health to the awesome-healthcare list: https://github.com/kakoni/awesome-healthcare/pull/106

This should help increase the project's visibility in the health informatics open-source community!
